README.rst

```diff
@@ -0,0 +1,101 @@
+Part of `edX code`__.
+
+__ http://code.edx.org/
+
+cookiecutter-django-ida
+=======================
+
+A cookiecutter_ template for edX Django projects.
+
+.. _cookiecutter: http://cookiecutter.readthedocs.org/en/latest/index.html
+
+
+This cookiecutter template is intended for new edX independently-deployable apps (IDAs). It includes the following packages:
+
+* Django 1.8.x
+* Django REST Framework
+* Django Waffle
+
+The necessary configuration is also in place to support:
+
+* i18n,
+* documentation (using Sphinx_),
+* authentication with edX Open ID Connect,
+* loading settings from YAML (for production),
+* Pylint/PEP8 validation,
+* and Travis CI.
```

Comment:
Sentence-cap items in this list and delete punctuation (and the initial "and" for the last item)